supervision/dataset/utils.py

```diff
@@ -3,7 +3,7 @@
 import random
 import shutil
 from pathlib import Path
-from typing import TYPE_CHECKING, Dict, List, Optional, Tuple, TypeVar, Union
+from typing import TYPE_CHECKING, Optional, TypeVar, Union
```

Comment:
lets replace `Union` with `|` and `Optional` with `| None` to keep things consistent